recipes/amdgpu/gcc11-hip53/config.yaml

```diff
@@ -0,0 +1,6 @@
+name: gcc-amdgpu
+store: /user-environment
+spack:
+    repo: https://github.com/spack/spack.git
+    commit: v0.20.1
```

Comment:
the `releases/v0.20` branch receives back ports of bug fixes but the recipes and interface are unchanged.